ddl/db_integration_test.go

```diff
@@ -15,7 +15,6 @@ package ddl_test
 
 import (
 	"fmt"
-
```

Comment:
Why this blank line has been deleted?
Shall we split the libs and third-party libs by blank line?

Reply:
Done.